### ISSUE TITLE
which version is the pytorch3d's ?

### ISSUE THREAD

**Issue by aurelianocyp** [MrTornado24/Next3D#35]
hello, I met a problem that the pytorch3d can not installed successfully, which version do you adopt

**Comment by MrTornado24** [comment]
Hi, I updated my environment and I use 0.7.4 now. Older versions may also work. How did you install it?

**Comment by aurelianocyp** [comment]
@MrTornado24 thanks for your reply, I have solved this problem, I use 0.7.5. my install is just like this:
```shell
git clone https://github.com/facebookresearch/pytorch3d.git
cd pytorch3d && pip install -e .
```
I just use Ubuntu 18.04 instead of 20.04, then, it succeeded.
